agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -78,6 +81,222 @@ namespace beerocks {
 
 const char *backhaul_manager::s_arrStates[] = {FOREACH_STATE(GENERATE_STRING)};
 
+/**
+ * @brief Gets media type group for given interface.
+ *
+ * @param[in] interface_name Name of the local interface.
+ * @param[in, out] media_type_group The media type group of the connecting interface.
+ *
+ * @return True on success and false otherwise.
+ */
+static bool get_media_type_group(const std::string &interface_name,
+                                 ieee1905_1::eMediaTypeGroup &media_type_group)
+{
+    // TODO: Currently, only Ethernet is supported
+    // When dealing with WiFi interfaces, use NL80211 command to know if media type group is WiFi
+    media_type_group = ieee1905_1::eMediaTypeGroup::IEEE_802_3;
```

Comment:
Which NL80211 command should be used to get the media type? Can you add this to the comment and perhaps a link to the task which will add it?

Reply:
Done